recipe/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "cytosnake" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/WayScience/CytoSnake/archive/v{{ version }}.tar.gz"
+  sha256: 39cc1d5284380492c7a1a41c7a84ee6269c0c2c57ec559a1c8af8a4a95b8f26b
+
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+  ignore_prefix_files:
+    - docs
+    - tests
+  run_exports:
+     - {{ pin_subpackage('cytosnake', max_pin="x.x") }}
+
+requirements:
+  build:
+    - python >=3.6, <=3.10
+    - pip
+    - setuptools
+  host:
+    - python >=3.6, <=3.10
+  run:
+    - python =3.10.*
+    - pip
+    - numpy >=1.17
```

Comment:
might cause some code decay for numpy >= 1.20 as functions change. If a range is allowed consider that instead.

Reply:
added! thanks